pkg/cluster/internal/providers/docker/network.go

```diff
@@ -18,7 +18,7 @@ package docker
 
 import (
 	"bytes"
-	"crypto/sha1"
+	"crypto/sha1" //nolint:gosec // used as a seed to subnet generator
```

Comment:
I think we should probably just disable this linter wholesale, I'm not aware of it catching much useful and it complains about a lot of intentional, well-meant behavior

Reply:
roger that